recipes/gvisor/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "gvisor" %}
+{% set version = "20200323.0" %}
+
+package:
+  name: go-{{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/google/{{ name }}/archive/release-{{ version }}.tar.gz
+  sha256: 4a4b0d42ef741cfdb831ebc05ef6f483217515f6d186abc96af6fea3d1312353
+  patches:
+    - syscalls_rmv_boottime.patch   # CLOCK_BOOTTIME is not a defined variable in time.h header of C Library in conda-forge. Remove from source code.
+
+build:
+  number: 0
+  skip: True  # [osx or win]
+  ignore_run_exports:
+    - libstdcxx-ng
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - {{ compiler('c') }}
+    - {{ compiler('go') }}
+    - git
+    - bazel
+  host:
+    - pip
+    - python
+  run:
+    - python
```

Comment:
is python a requirement? 
```suggestion
```

Reply:
yes it's listed as a requirement on https://github.com/google/gvisor under installing from source